ocean-client/addons/command-center/src/commands/set-pair.ts

```diff
@@ -0,0 +1,51 @@
+import { Bot } from '../utils/available-bot'
+import { fetchListOfPoolPairs } from '../utils/helpers'
+import { Command, Commands } from './command'
+
+export class SetPair extends Command {
+  private pair: string | undefined
+  private listOfPairs: string[] = []
+  private static usageMessage: string = Commands.SetPair + ' BTC-DFI'
+
+  static description = 'Sets given pair as new lm-reinvest pair.\n' + SetPair.usageMessage
+
+  static descriptionFor(bots: Bot[]): string | undefined {
+    if (!bots.includes(Bot.REINVEST)) return undefined
+    return SetPair.description
+  }
+
+  async prepare() {
+    this.listOfPairs = await fetchListOfPoolPairs()
```

Comment:
if my listOfPoolPairs returns the list of pools pairs (and not as it is right now a list of dtokens), then this is fine.